pkg/apis/volumesnapshot/v1beta1/types.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package v1alpha1
+package v1beta1
 
 import (
```

Comment:
Verify that CRDs actually support defaulting, and make sure it works.

Reply:
If defaulting is not supported we need to implement WebHooks for verification. Alternatively you could make the field required and add validation to verify values.